examples/solids/src/setuplibceed.c

```diff
@@ -37,29 +37,35 @@ problemData problemOptions[3] = {
     .setupgeo = SetupGeo,
     .apply = LinElasF,
     .jacob = LinElasdF,
+    .energy = LinElasEnergy,
     .setupgeofname = SetupGeo_loc,
     .applyfname = LinElasF_loc,
     .jacobfname = LinElasdF_loc,
+    .energyfname = LinElasEnergy_loc,
     .qmode = CEED_GAUSS
   },
   [ELAS_HYPER_SS] = {
     .qdatasize = 10,
     .setupgeo = SetupGeo,
     .apply = HyperSSF,
     .jacob = HyperSSdF,
+    .energy = LinElasEnergy,
     .setupgeofname = SetupGeo_loc,
     .applyfname = HyperSSF_loc,
     .jacobfname = HyperSSdF_loc,
+    .energyfname = LinElasEnergy_loc,
```

Comment:
These should use HyperSSEnergy and we should write the strain energy functions into the documentation as well. @ArashMehraban This would be a great contribution from you.

Reply:
@jedbrown Just sent it to Jeremy to upload. Look forward to comments

Reply:
I'm a bit confused on why not push it yourself? We can help you and revise it, but you will keep authorship of your contribution